pulp_rpm/tests/functional/conftest.py

```diff
@@ -114,8 +114,51 @@ def rpm_content_distribution_trees_api(rpm_client):
     return ContentDistributionTreesApi(rpm_client)
 
 
+@pytest.fixture(scope="session")
+def rpm_copy_api(rpm_client):
+    return RpmCopyApi(rpm_client)
+
+
+@pytest.fixture(scope="class")
+def rpm_repository_factory(rpm_repository_api, gen_object_with_cleanup):
```

Comment:
By following pulp_file tests, I wrote this factory. Due to that, I also updated a couple of already migrated tests, like test_sync, and test_publish.

Reply:
If you can split that into a separate commit that would be good.

Reply:
Sure, will do that.